frontend/fingertips-frontend/components/viewPlots/OneIndicatorOneAreaViewPlots/index.tsx

```diff
@@ -68,6 +68,12 @@ export function OneIndicatorOneAreaViewPlots({
                     searchState={searchState}
                     groupIndicatorData={groupData}
                     xAxisTitle="Year"
+                    yAxisTitle={
+                      indicatorMetadata?.unitLabel
+                        ? `Value: ${indicatorMetadata?.unitLabel}`
+                        : ''
+                    }
+                    measurementUnit={indicatorMetadata?.unitLabel ?? ''}
```

Comment:
As with the others, I think `measurementUnit` and in this case `yAxisTitle` should be optional properties:
```suggestion
                    yAxisTitle={
                      indicatorMetadata?.unitLabel
                        ? `Value: ${indicatorMetadata?.unitLabel}`
                        : undefined
                    }
                    measurementUnit=indicatorMetadata?.unitLabel
```

Reply:
I think this one is still outstanding.